bracketed-paste.el

```diff
@@ -128,7 +128,8 @@
   '(suspend-hook suspend-tty-functions kill-emacs-hook delete-frame-functions))
 
 (defun bracketed-paste--safe-tty-state-call (terminalish send-tty)
-  (cond ((null terminalish) (funcall send-tty))
+  (cond ((window-system) nil) ; don't call send-tty if not a TTY
```

Comment:
> Use of this function as a predicate is deprecated.  Instead,
use ‘display-graphic-p’ or any of the other ‘display-*-p’
predicates which report frame’s specific UI-related capabilities.